docs/posts/roadmap-2024-H1/index.qmd

```diff
@@ -0,0 +1,381 @@
+of ordered results and the index. pandas is implemented for single-threaded
+execution and has a lot of baggage when it comes to distributed execution. While
+projects like Modin and pandas on Spark (formerly Koalas) attempt to scale the
+pandas API, any project that attempts the feat is doomed to a dubious support
+matrix of operations.
+
+Instead, Wes McKinney envisioned Ibis as a portable Python dataframe where the
+API is decoupled from the execution engine. Ibis code scales to the backend it
+is connected to. Any other Python dataframe library locks you into its execution
+engine. While they may claim to be easy to migrate to, this is rarely the case.
+The founders of Voltron Data experienced these pains with the pandas API
+themselves in previous efforts, including cuDF. For Theseus and as an
+open-source standard, we believe Ibis is the right approach.
+
+Instead of using Snowpark Python for Snowflake, you can use Ibis on Snowflake.
+Instead of using PySpark or pandas on Spark, you can use Ibis on Spark. Instead
+of using the pandas API on BigQuery (built on top of Ibis), you can use Ibis on
+BigQuery. Instead of using PyStarburst on Starburst Galaxy, you can use Ibis on
+Starburst Galaxy. Instead of using the Polars Python on the Polars execution
+engine, you can use Ibis on Polars. Instead of using DataFusion Python on
+DataFusion execution engine, you can use Ibis on DataFusion. Instead of
+executing SQL strings on DuckDB through the Python client, you can use Ibis on
+DuckDB. And so on...
+
+Ibis brings a Python dataframe interface to data platforms that only have SQL,
+and brings a standard Python dataframe interface to data platforms that have
+their own Python dataframe interface. It is the only portable Python dataframe
+that can serve as a standard across the data ecosystem.
+:::
+
+Voltron Data supports Ibis for the same reason it supports Apache Arrow.
+Open-source standards make it easier to build the composable data ecosystem and
+reduce friction for data teams looking to swap out components as their needs
+change. If you're already using Ibis as your frontend, Apache Arrow for data
+interchange, and Substrait for intermediary representation, you can swap in
+Theseus with ease!
+
+::: {.callout-important}
+[Ibis is independently governed](https://github.com/ibis-project/governance) and
+not owned by Voltron Data. While currently four out of five members of the
+steering committee are employed by Voltron Data (the fifth being at Alphabet
+working on Google BigQuery), we expect this to change over time as more
+organizations join the project. **Ibis will never be solely available for VoDa
+products.**
+
+Voltron Data also welcomes this dilution of power and influence! A healthy
+open-source project is one that is not controlled by a single entity. This is
+true of [Apache Arrow](https://arrow.apache.org) and other open-source projects
+that Voltron Data employees have been instrumental in building.
```

Comment:
```suggestion
not owned by Voltron Data. Currently, four out of five members of the
steering committee are employed by Voltron Data (the fifth being at Alphabet
working on Google BigQuery). We intend to work toward a more diverse
representation of companies and organizations as the project continues to grow.
```

Reply:
We should say "are working toward" instead of "intend to work toward".